### DISCUSSION TITLE
Smart Connections stuck in embedding process

### DISCUSSION THREAD

**Discussion by sjhogerzeil** [brianpetro/obsidian-smart-connections#855] [Q&A]
![20241027-000238](https://github.com/user-attachments/assets/772862da-c01d-454e-baa1-d5c8e2df5aa4)

Using TaylorAl/bge-micro-v2 to embed locally, the files are loaded correctly after installing Smart Connections, but the embedding process gets stuck without making progress

On the console, I get these error messages:

Help would be greatly appreciated!

**Comment by brianpetro** [comment]
Thanks for the screenshots 

Unfortunately those ones don't tell me much about what's going on

Do you see any other errors that can be screenshotted?

 🌴

**Comment by sjhogerzeil** [comment]
Hi Brian,

Thanks for your time.

Sorry to have bothered you. I resolved the issue (it plagued me for 
hours) by de-installing, re-installing, re-loading etc. Suddenly, it 
worked…

Have a great day, I love your work, thank you very much!

Regards,

Simon
